examples/model_configs/vllm_model_config.yaml

```diff
@@ -1,13 +1,33 @@
-model:
-  base_params:
-    model_args: "pretrained=HuggingFaceTB/SmolLM-1.7B-Instruct,revision=main,dtype=float16,gpu_memory_utilization=0.6,max_num_seqs=1,max_num_batched_tokens=4096"
-  generation:
-    temperature: 0.3
-    repetition_penalty: 1.0
-    frequency_penalty: 0.0
-    presence_penalty: 0.0
-    seed: 42
-    top_k: -1
-    min_p: 0.0
-    top_p: 0.9
-    max_new_tokens: 2048
+model_parameters:
+  model_name: "HuggingFaceTB/SmolLM2-1.7B-Instruct"
+  revision: "57aa3c6599c53705406c648e7acca7e11dc45ea3"
+  dtype: "float16"
+  tensor_parallel_size: 1
+  data_parallel_size: 1
+  pipeline_parallel_size: 1
+  gpu_memory_utilization: 0.6
+  max_model_length: null
+  swap_space: 4
+  seed: 42
+  trust_remote_code: False
+  use_chat_template: True
+  add_special_tokens: True
```

Comment:
a group of params for preprocessing could be interesting (multichoice continutation start space, pairwise tk, add special tokens, use chat templates, ...)